clang-tools-extra/test/clang-tidy/checkers/google/readability-todo.cpp

```diff
@@ -24,3 +24,7 @@
 // TODO(b/12345): find the holy grail
 // TODO (b/12345): allow spaces before parentheses
 // TODO(asdf) allow missing semicolon
+// TODO: bug 12345678 - Remove this after the 2047q4 compatibility window expires.
+// TODO: example.com/my-design-doc - Manually fix up this code the next time it's touched.
+// TODO(bug 12345678): Update this list after the Foo service is turned down.
+// TODO(John): Use a "\*" here for concatenation operator.
```

Comment:
Given that this is the text from the style-guide, I'd be concerned regarding licensing? It's under CC 3.0
You could just rewrite the lines that were already here to fit the pattern

Reply:
Great point, done.